static_src/util/cf_api.js

```diff
@@ -47,7 +46,9 @@ export default {
     return resources.map((r) => this.formatSplitResponse(r));
   },
 
-  fetch(url, action, multiple, ...params) {
+  fetch(url, _action, multiple, ...params) {
+    // Set a default noop action handler
+    const action = typeof _action === 'function' ? _action : () => {};
```

Comment:
Sure, I wouldn't say "dangerous" but yes, there would be no feedback if you didn't include the action callback. This is just like with promises, you wouldn't get any feedback if you forgot to call `.this` on your promise.

I see this method as going away. It's useful to have the fetch utilities to handle multiple and paged responses, but mostly I think we should just use the request promise for handling any followup action rather than in this utility. So the intention is to move to a model where there is less feedback for developer errors -- a promise-based model which I think is familiar to javascript developers.